client/landing/stepper/declarative-flow/internals/steps-repository/playground/lib/resolve-blueprint-from-url.ts

```diff
@@ -0,0 +1,222 @@
+import { Blueprint } from '@wp-playground/client';
+
+/**
+ * All code in this file is copied from @wp-playground/website package
+ *
+ * TODO: Once this function is moved to a package that's supposed to be published,
+ * we can switch to using that and delete this file here.
+ */
```

Comment:
We could export these in the `@wp-playground/client` package.

Reply:
Yep! I need to build more context before I can start thinking properly around refactoring stuff in playground packages.

Reply:
Let me know if you need any help.